docs/guides/key-manager/upgrade-lsp6.md

```diff
@@ -0,0 +1,246 @@
+---
+sidebar_label: 'Upgrade LSP6 Key Manager'
+sidebar_position: 4
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Upgrade the LSP6 Key Manager
+
+:::info Requirements
+
+You will need a Universal Profile that you can control via its KeyManager to follow this guide.
```

Comment:
What about adding some internal links?

```suggestion
You will need a Universal Profile that you can control via its [KeyManager](../../../standards/universal-profile/lsp6-key-manager) to follow this guide.
```

(Please check the number of `../` i did it from GitHub im not sure if it is correct)